mythtv/libs/libmythtv/mythplayeroverlayui.cpp

```diff
@@ -230,6 +231,7 @@ void MythPlayerOverlayUI::UpdateSliderInfo(osdInfo &Info, bool PaddedFields)
         Info.text[relPrefix + "description"] = desc;
         Info.text[relPrefix + "playedtime"] = text1;
         Info.text[relPrefix + "totaltime"] = text2;
+        Info.text[relPrefix + "wallclocktime"] = wallclock.toLocalTime().toString("ddd MMM d yyyy, hh:mm:ss");
```

Comment:
Not sure "wallclocktime" is the right name for this data field, as that implies the time right now (whenever that is) on the clock on the wall when you're watching the recording. Maybe "recordingtime" or "recordedtime" or "broadcasttime" or something like that.

Reply:
Done.  "wallclocktime" was inherited from #127 and meant "what the wall clock said at this time in the recording".  But I agree with you and went with "recordedtime".  Also edited the title of this pull request to match.

Please close #127 as it is obsolete.